dashboard/api_test.py

```diff
@@ -387,9 +423,59 @@ def test_not_paid_not_passed(self):
             enr_end=self.now-timedelta(weeks=53),
             edx_key="course-v1:MITx+8.MechCX+2014_T1"
         )
+        with patch('courses.models.CourseRun.is_upgradable', new_callable=PropertyMock) as upgr_mock, patch(
+            'courses.models.CourseRun.has_frozen_grades', new_callable=PropertyMock
+        ) as froz_mock:
+            upgr_mock.return_value = is_upgradable
+            froz_mock.return_value = has_frozen_grades
+            run_status = api.get_status_for_courserun(crun, self.mmtrack)
+        assert run_status.status == expected_status
+        assert run_status.course_run == crun
+
+    @patch('grades.models.FinalGrade.objects.get', autospect=True)
+    @patch('courses.models.CourseRun.is_upgradable', new_callable=PropertyMock)
+    @patch('courses.models.CourseRun.has_frozen_grades', new_callable=PropertyMock)
+    def test_not_paid_in_past_grade_frozen(self, upgr_mock, froz_mock, final_grades_mock):
```

Comment:
I might be reading this wrong but aren't `upgr_mock` and `froz_mock` in the wrong order?